openssl 1.1.1q has a problem where a `memcmp` call is not declared before its
usage in a test file. due to this, MacOS is not able to compile the test for
1.1.1q without screwing around with compiler flags. this patch introduces the
`memcmp` declaration
--- test/v3ext.c
+++ test/v3ext.c
@@ -8,6 +8,7 @@
  */
 
 #include <stdio.h>
+#include <string.h>
 #include <openssl/x509.h>
 #include <openssl/x509v3.h>
 #include <openssl/pem.h>
